### DISCUSSION TITLE
Port forward issue on running docusaurus project live preview

### DISCUSSION THREAD

**Discussion by kazmi066** [community/community#37981] [Codespaces]
I am running a simple project of Docusaurus with `npm start` which runs successfully.
Using Github Codespaces to run the project but the forwarded route doesn't seem to be working.

Attached the screenshots of environment.
[Running Ports](https://i.stack.imgur.com/ytRwK.png)
[Terminal](https://i.stack.imgur.com/nVx47.png)
[Opened URL](https://i.stack.imgur.com/q592m.png)

Already posted on stackoverflow but not found any solution

**Comment by fara-nak** [comment]
hi @kazmi066  sorry to hear this, could you please send us your codespace name or id aong with the log file by going to command pallet and typing "codespaces: export logs"

**Reply by kazmi066** [reply]
Codespace name: shiny bassoon
[logs.zip](https://github.com/community/community/files/9912423/logs.zip)

**Reply by fara-nak** [reply]
thanks for the logs, we had an internal issue earlier today which might be related to this. could you please stop your codespace and restart it and try running your app again and let us know if you still see the issue?

**Reply by kazmi066** [reply]
Nope, I stopped the codespace and restarted it but issue is still there :p

**Reply by kazmi066** [reply]
I deleted the codespace just now and tried creating a new one but still same.
Codespace name: `laughing space disco`

**Reply by fara-nak** [reply]
interesting we created a codespace in your region with this repo(micrsoft/vscodespaces-quickstart) and we didn't see any issue. it's a piblic repo  in case you want to test and see if it works for you.
can you please share the logs for this new codespace?

**Reply by lgiancol** [reply]
Hi, the Codespace name will actually be something like `kazmi066-laughing-space-disco-<random-string>`.

If you are using VS Code in the browser, it will be the start of the URL, before the `.github.dev`.

If you are using VS Code desktop, you can:
1. Open VS Code settings
2. Make sure the `Codespaces: Show Performance Explorer` checkbox is checked
3. Open up `Remote Explorer` tab in VS Code. There should be a section at the bottom called, "Codespace Performance"
4. Copy the `Codespace ID` from that view and send it here

We can get the Codespace name using that ID.

**Reply by kazmi066** [reply]
Hi, thanks for replying. Here is the info
Codespace name: https://kazmi066-laughing-space-disco-w9479q4566qhp9g.github.dev/
Codespace ID: f2621849-834b-499c-909d-908ae217a250

Attached new logs
[logs.zip](https://github.com/community/community/files/9926636/logs.zip)

Also codespace is working fine for another React based project of mine...so region might not be the issue
